views/drones/list.hbs

```diff
@@ -1 +1,27 @@
-<h2>Available drones</h2>
\ No newline at end of file
+<h2>Available drones</h2>
+{{#each dronesFromDB}}
+
+
+<div class="container">
+     <div class="row">
+         <div class="col">
+             <div class="card" style="width: 18rem;">
+        <div class="card-body" style="width: 18rem;">
+            <h5 class="card-title">{{name}}</h5>
+            <h6 class="card-subtitle mb-2 text-muted"><i>Maximum Speed: </i>{{this.maxSpeed}} m/s</h6>
+            <p class="card-text"><i>Propellers: </i>{{this.propellers}}</p>
+
+            <div class="btn-container">
+                <a href="/drones/{{this._id}}/edit" class="btn btn-secondary">Edit</a>
+                <form action="/drones/{{id}}/delete" method="POST">
+                    <button type="submit" class="btn btn-danger"> Delete Drone </button>  
+                </form>
+            </div>
+        </div>
+    </div>
+         </div>
+     </div>
+</div>
+
+{{/each}} 
+<a href="/drones/create" class="btn btn-warning">Add Drones</a>
```

Comment:
I would name this button "Add drone" (singular instead of plural).

Reply:
noted